### DISCUSSION TITLE
is there a hiring space in github

### DISCUSSION THREAD

**Discussion by wingedping** [community/community#47184] [New to GitHub]
Hi, Im learning to code react native to create an MVP of my product but seriously it's going to take me too long. I have 50k lines of code but really need someone to help me debug issues to get it even at demo quality is there anyone in this space to talk to?

**Comment by Akash1134** [comment]
Hi @wingedping  ,

we're happy you're here 😀

However, we currently do not allow job listings or hiring on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
